wp-content/themes/core/components/pagination/single/single.php

```diff
@@ -0,0 +1,35 @@
+<?php
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$c = \Tribe\Project\Templates\Components\pagination\single\Controller::factory( $args );
+
+?>
+<?php if ( ! empty( $c->previous_post ) || ! empty( $c->next_post ) ) { ?>
+	<nav <?php echo $c->wrapper_classes(); ?><?php echo $c->wrapper_attrs(); ?> >
+
+		<h3 <?php echo $c->header_attrs(); ?><?php echo $c->header_classes(); ?>>
+			<?php esc_html_e( 'Post Pagination', 'tribe' ); ?>
+		</h3>
+
+		<ol <?php echo $c->container_classes(); ?><?php echo $c->container_attrs(); ?>>
```

Comment:
Will this always be ol? Do we need an option where this is an ul?

Reply:
Nope, will always be ol